presto-native-execution/presto_cpp/main/operators/LocalPersistentShuffle.h

```diff
@@ -132,6 +133,10 @@ class LocalPersistentShuffleReader : public ShuffleReader {
 
   velox::BufferPtr next(bool success) override;
 
+  folly::F14FastMap<std::string, int64_t> stats() const override {
+    return {{"local.read", 123}};
```

Comment:
could we just use `{"local.read", 0}` instead of a 123 ? Someone without context may misinterpret the 123 as something meaningful.

Reply:
Sure. I can also comment that this is a fake stat for testing purpose.